tox.ini

```diff
@@ -12,6 +13,7 @@ commands=
 deps=
   dj111: Django>=1.11,<1.12
   dj21: Django>=2.1,<2.2
+  dj30: Django>=3.0,<4.1
```

Comment:
This includes 4.0. Shouldn't it be `Django>=3.0,<3.1`?

Reply:
true, fixed.